helm/openidm/values.yaml

```diff
@@ -34,7 +34,7 @@ component: openidm
 
 image:
   repository: forgerock-docker-public.bintray.io/forgerock/idm
-  tag: 7.0.0-M2
+  tag: 7.0.0-SNAPSHOT
```

Comment:
IDM doesn't have SNAPSHOT builds of docker images.  Will this cause an issue?

Reply:
IDM have one https://forgerock.bintray.com/public/forgerock/idm/

In fact for now this `7.0.0-SNAPSHOT` tag is used by all the products to point to the latest tag more or less.
This is something we have to look and found a better for this tag.